### PR TITLE
Setup so acceptance and create property page acceptance

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
use funkie and funkie-phantom
change urls
and .editorconfig

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +34,11 @@
     "browserify": "^13.3.0",
     "eslint": "^3.14.0",
     "eslint-config-homeoffice": "^2.1.0",
+    "funkie": "0.0.5",
+    "funkie-phantom": "0.0.1",
     "nodemon": "^1.11.0",
-    "npm-run-all": "^4.0.1"
+    "npm-run-all": "^4.0.1",
```
Comment (lennym) [inline package.json]: I know it isn't *actually* part of this diff, but this doesn't look to be being used, so can (and should) be removed.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "build:translate": "hof-transpiler ./apps/**/translations/src -w --shared ./node_modules/hof-template-partials/translations",
     "test": "npm run test:lint",
     "test:lint": "eslint .",
+    "test:acceptance": "funkie so-acceptance --steps",
```
Comment (lennym) [inline package.json]: You might want to add this to your `npm test` script.

**Comment by lennym** [comment]
I'd quite like to see https://www.npmjs.com/package/hof-util-autofill used to get to test steps instead of session hacking as it makes for more resilient tests, but not strongly enough to block merging.

If you'd like to use it then feel free.

File: package.json (hunk 1 above)
Comment (sulthan-ahmed) [inline package.json]: Unless my eyes are deceiving me, I'm using it here https://github.com/UKHomeOffice/right-to-rent-check/pull/13/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L5

File: package.json (hunk 2 above)
Comment (sulthan-ahmed) [inline package.json]: good spot

**Comment by sulthan-ahmed** [comment]
I didn't realise we are hacking the session, where are we hacking it? Is it in the Before step to get to a particular step?   So does hof-util-autofill actually go through the steps properly?

Of course, I'll use the hof-util-autofill @lennym.  If you don't mind I'll put it into another PR

**Comment by lennym** [comment]
Yeah, the `visitPage` method used in the `before` modifies the session to allow the test to skip directly to a step, but it means a) that the tests doesn't match real user actions, and b) the tests can't be run against other environments.

Happy to wait on it though.

File: package.json (hunk 1 above)
Comment (lennym) [inline package.json]: You are correct. My bad.

Comment (sulthan-ahmed) [inline package.json]: no problem, you are human after all 👍

**Comment by sulthan-ahmed** [comment]
as always valid points @lennym to use hof-util-autofill

**Pull request merged**